### ISSUE TITLE
Wrong balance amount

### ISSUE THREAD

**Issue by nikaro** [bnjbvr/kresus#415]
Hello,

I have wrong balance amount for all of my account, while it doesn't seem to have operations fetching problem.

```
[2016-05-13 11:16:12:577] info - sources/weboob | Fetch started: running process ./weboob/scripts/operations.sh...
[2016-05-13 11:16:16:922] info - sources/weboob | weboob exited with code 0
[2016-05-13 11:16:16:924] info - sources/weboob | OK: weboob exited normally with non-empty JSON content.
[2016-05-13 11:16:17:990] info - accounts-manager | Updating 'last checked' for linked accounts...
[2016-05-13 11:16:18:2] info - accounts-manager | Informing user new operations have been imported...
[2016-05-13 11:16:18:2] info - accounts-manager | Checking alerts for accounts balance...
[2016-05-13 11:16:18:2] info - accounts-manager | Checking alerts for operations amount...
[2016-05-13 11:16:18:3] info - accounts-manager | Post process: done.
```

I have no idea where does the problem come from, maybe a bug due to the fact that i have two bank accounts in the same bank ? :-/

Let me know which information do you need to investigate further.

**Comment by bnjbvr** [comment]
Hi! Thanks for the report. Let's try the usual suspects:

- do you have two accounts which have the same number or the same name?
- have you checked the duplicates screen? Are there duplicates that aren't supposed to show up there? Are there duplicate transactions that should appear but don't?
- have you tried changing the duplicate threshold, in the preferences?
- did your bank suddenly show much older transactions?
- was a new account of yours recently imported in Kresus?

**Comment by nikaro** [comment]
Thanks for the answer :-)

> do you have two accounts which have the same number or the same name?

Nope.

> have you checked the duplicates screen? Are there duplicates that aren't supposed to show up there? Are there duplicate transactions that should appear but don't?

No duplicates (nor on the duplicates screen or in the reports).

> have you tried changing the duplicate threshold, in the preferences?

I tried to set the threshold at 48h, but it only detected a false positive.

> did your bank suddenly show much older transactions?

I can see transactions aged from 04/2015, i don't remember it was the case before (i had only 2 months old transactions i think).

> was a new account of yours recently imported in Kresus?

Nope.

**Comment by bnjbvr** [comment]
Thank you! That's probably #413 then. I'll make sure it gets fixed and that #190 is implemented as soon as possible.

**Comment by nikaro** [comment]
Great, thank you :-)
